.github/workflows/irc-notifications.yaml

```diff
@@ -107,6 +107,41 @@ jobs:
             ${{ env.SUMUP }}\n\
             ${{ github.event.compare }}"
 
+      - name: Shorten PR fields
+        if: github.event_name == 'pull_request'
+        env:
+          TITLE: ${{ github.event.pull_request.title }}
+          BODY:  ${{ github.event.pull_request.body }}
+        run: |
+          # -------------------------------------
+          echo "# script parse.pl"
+          cat <<'EOS' > parse.pl
+          use v5.14; use strict; use warnings;
+          my ( $key, $len ) = @ARGV;
+          $key //= ''; $len //= 50;
+          my $txt = substr( $ENV{$key} || "Unknown $key", 0, $len );
+          $txt =~ s{\\n}{\n}g; $txt =~ s{\\t}{ }g; $txt =~ s{\t}{ }g;
+          $txt =~ s{^EOS}{}mg;
```

Comment:
It seems like an attempt to avoid premature termination of the output by removing the string used as a delimiter. But the delimiter used to wrap the output of this string is `EOF`, not `EOS`.